nixos/modules/programs/xfconf.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.xfconf;
+
+in {
+  meta = {
+    maintainers = teams.xfce.members;
+  };
+
+  options = {
+    programs.xfconf = {
```

Comment:
I think it's better if you add a `package` option that defaults to `pkgs.xfce.xfconf`, then pass that result to systemPackages and dbus packages.

Reply:
Do you envision a situation where one would want to override `xfconf` for the module but not for the rest of the system the way overlays do?

Reply:
Being used by many components of Xfce, it is sensible that an override of `xfconf`, if needed, would be better done system wide with overlays, for instance. So I would keep it simple and not add a `package` option.